src/views/skuLookup.tsx

```diff
@@ -0,0 +1,349 @@
+      [SKU_INVENTORY.COUNTRY]: "",
+      [SKU_INVENTORY.STOCK_LEVEL]: "",
+    },
+  ]);
+  const { width } = useScreenSize();
+
+  useEffect(() => {
+    setStatesData();
+  }, []);
+
+  const setStatesData = (catalog: any = {}) => {
+    setSkuDetailsTable([
+      {
+        [SKU_DETAILS.SKU]: catalog?.skuId?.toString() || "",
+        [SKU_DETAILS.DISPLAY_NAME]: catalog?.product?.displayName || "",
+        [SKU_DETAILS.BRAND]: catalog?.product?.brand?.displayName || "",
+      },
+      {
+        [SKU_DETAILS.EFFECTIVELY_ACTIVE]:
+          catalog?.effectivelyActive?.toString() || "",
+        [SKU_DETAILS.START_DATE]: catalog?.startDate || "",
+        [SKU_DETAILS.END_DATE]: catalog?.endDate || "",
+      },
+      {
+        [SKU_DETAILS.SKU_STATE]: catalog?.skuState?.toString() || "",
+        [SKU_DETAILS.RESERVATION_ENABLED]:
+          catalog?.reservationEnabled?.toString() || "",
+        [SKU_DETAILS.STORE_ONLY]: catalog?.storeOnly?.toString() || "",
+      },
+    ]);
+
+    setPriceDetailsTable([
+      {
+        [SKU_PRICE_DETAILS.US_LIST_PRICE]:
+          catalog?.listPriceUS?.toString() || "",
+        [SKU_PRICE_DETAILS.US_SALE_PRICE]:
+          catalog?.salePriceUS?.toString() || "",
+        [SKU_PRICE_DETAILS.CA_LIST_PRICE]:
+          catalog?.listPriceCA?.toString() || "",
+        [SKU_PRICE_DETAILS.CA_SALE_PRICE]:
+          catalog?.salePriceCA?.toString() || "",
+      },
+    ]);
+
+    setAtgInventory([
+      {
+        [SKU_INVENTORY.COUNTRY]: catalog?.inventory?.[0]?.country || "",
+        [SKU_INVENTORY.STOCK_LEVEL]:
+          catalog?.inventory?.[0]?.stockLevel?.toString() || "",
+        [SKU_INVENTORY.ON_HOLD]:
+          catalog?.inventory?.[0]?.onHold?.toString() || "",
+      },
+      {
+        [SKU_INVENTORY.COUNTRY]: catalog?.inventory?.[1]?.country || "",
+        [SKU_INVENTORY.STOCK_LEVEL]:
+          catalog?.inventory?.[1]?.stockLevel?.toString() || "",
+        [SKU_INVENTORY.ON_HOLD]:
+          catalog?.inventory?.[1]?.onHold?.toString() || "",
+      },
+    ]);
+
+    setProductInfo([
+      {
+        [SKU_PRODUCT_INFO.PRODUCT]: catalog?.product?.productId || "",
+        [SKU_PRODUCT_INFO.ACTIVATION_DATE]:
+          catalog?.product?.activationDate || "",
+        [SKU_PRODUCT_INFO.END_DATE]: catalog?.product?.endDate || "",
+      },
+      {
+        [SKU_PRODUCT_INFO.URL]: catalog?.product?.productUrl || "",
+        [SKU_PRODUCT_INFO.EFFECTIVELY_ACTIVE]:
+          catalog?.product?.isEffectivelyActive?.toString() || "",
+        [SKU_PRODUCT_INFO.ENABLED]: catalog?.product?.enabled?.toString() || "",
+      },
+    ]);
+  };
+
+  const setYantriksData = (inv: any = {}, country: string) => {
+    const inventory = {
+      [SKU_INVENTORY.COUNTRY]: country || "",
+      [SKU_INVENTORY.STOCK_LEVEL]:
+        inv.availabilityByProducts?.[0]?.availabilityDetails?.atp?.toString() ||
+        "",
+    };
+    let firstEmptyCountry = yantriksInventory.findIndex((inventory) => {
```

Comment:
for a single line arrow function, `{` and `}` are not needed.
you can directly add the next line.